applications/StructuralMechanicsApplication/custom_constitutive/linear_plane_stress.cpp

```diff
@@ -27,15 +26,15 @@ namespace Kratos
 //************************************************************************************
 
 LinearPlaneStress::LinearPlaneStress()
-    : ConstitutiveLaw()
+    : ElasticIsotropic3D()
```

Comment:
why are u deriving from the 3D one? i would not do this... the flatter the hierarchy the easier to read

Reply:
We could move the common functionalities to a utilities-class to stick with the flatter hierarchie

Reply:
In this specific case, since the shared part is minimal i would just reimplement it. I don't have a strong opinion on this though, so if you think it is important to derive it since it simplifies a lot then i am ok with it.

Reply:
I agree, flatter is better but, a lot of code was duplicated.
In order to make the code easier to read, I can ad a virtual class -- ElasticLaw -- and move the common functionalities inside.
Thus, 3D and 2D will be on the same level